CONTRIBUTING.md

```diff
@@ -72,9 +72,16 @@ Basic rules:
 
 ### Before landing
 
-* Pull Requests must be open for at least 48 hours unless changes include errata fixes, infrastructure maintenance, or tests
+* Pull Requests must be open for at least 48 hours.
+  * Pull Requests might be fast-tracked to then 8 hours with at least one approval if they're bug fixes, pre-discussed content-removal, or general content update that had no prior objections before the creation of the PR.
```

Comment:
I think it's better to have only 2 ways, 48 hours or fast track (immediate). Something in the middle, 8 hours, will be hard to manage and create more confusion.

Reply:
That's a good point. So having only the 48-hour and fast track? What would be a reasonable time to get a fast-tracked PR merged after any approval?

Reply:
you can take example from here:https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#fast-tracking-prs

Reply:
The example assumes an issue is already open. Should we move in the direction of having more defined issues before creating the PR's? A lot of our work is we see an issue and then create the PR with an explanation of what was fixed, changed, or added.

Reply:
> The example assumes an issue is already open. Should we move in the direction of having more defined issues before creating the PR's? A lot of our work is we see an issue and then create the PR with an explanation of what was fixed, changed, or added.

True. There are also PRs that don't explicitly have an issue but are part of the project's goal.